samples/materials/sandboxLit.mat

```diff
@@ -37,6 +39,8 @@ material {
 fragment {
     void material(inout MaterialInputs material) {
         prepareMaterial(material);
+        material.absorption = -log(vec3(1.0, 0.7, 0.1)) / 1.0;
```

Comment:
Is the log a particular formula or just a test?

Reply:
it's the formula from the disney paper. We'll have a helper somewhere where you give the "color at a distance".